.github/workflows/unittests_windows.yml

```diff
@@ -0,0 +1,31 @@
+name: Unittests
```

Comment:
Can we change this to `Unittests Windows` ?

Reply:
Was on it while you reviewed :D